lib/src/toggle-group/ToggleGroup.tsx

```diff
@@ -116,6 +117,7 @@ const Label = styled.label`
   font-style: ${(props) => props.theme.labelFontStyle};
   font-weight: ${(props) => props.theme.labelFontWeight};
   line-height: ${(props) => props.theme.labelLineHeight};
+  margin-bottom: ${(props) => !props.helperText && props.theme.containerMarginTop};
 `;
```

Comment:
Same here. In the specs these margins are shown as 0.25 rem. Is this change really needed and the specs should be updated?
![image](https://user-images.githubusercontent.com/10975289/171836134-a54e19ac-774d-4486-a39b-90ad943ae6ed.png)